engine/apps/alerts/migrations/0028_drop_alertreceivechannel_restricted_at.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.20 on 2023-07-26 06:41
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("alerts", "0027_remove_alertreceivechannel_restricted_at_from_state"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="ALTER TABLE alerts_alertreceivechannel DROP COLUMN restricted_at",
+            reverse_sql=migrations.RunSQL.noop
```

Comment:
Adding a reverse operation so the migration is reversible (just in case).

```suggestion
            reverse_sql="ALTER TABLE alerts_alertreceivechannel ADD COLUMN restricted_at datetime(6) NULL"
```

SQL generated by running `python manage.py sqlmigrate alerts 0014`

Reply:
thanks for that 👍